config/ohd.yml

```diff
@@ -3,24 +3,26 @@
 idspace: OHD
 base_url: /obo/ohd
 
-base_redirect: https://github.com/alanruttenberg/ohd-ontology
+base_redirect: https://github.com/oral-health-and-disease-ontologies/ohd-ontology
 
 products:
-- ohd.owl: https://raw.githubusercontent.com/alanruttenberg/ohd-ontology/master/src/ontology/ohd-2016-06-27-merged-inferred.owl
+- ohd.owl: https://raw.githubusercontent.com/oral-health-and-disease-ontologies/ohd-ontology/master/ohd.owl
+- ohd-base.owl: https://raw.githubusercontent.com/oral-health-and-disease-ontologies/ohd-ontology/master/ohd-base.owl
```

Comment:
This `ohd-base.owl` should use an `exact` entry to match its Ontology IRI: <http://purl.obolibrary.org/obo/ohd/ohd-base.owl>. I can fix that.